module/zfs/vdev.c

```diff
@@ -4623,8 +4623,8 @@ module_param(vdev_ms_count_limit, int, 0644);
 MODULE_PARM_DESC(vdev_ms_count_limit,
 	"Practical upper limit of total metaslabs per top-level vdev");
 
-module_param(zfs_delays_per_second, uint, 0644);
-MODULE_PARM_DESC(zfs_delays_per_second, "Rate limit delay events to this many "
+module_param(zfs_slow_ios_per_second, uint, 0644);
+MODULE_PARM_DESC(zfs_slow_ios_per_second, "Rate limit delay events to this many "
 	"IO delays per second");
```

Comment:
What would you think about renaming this to indicate that it's a limit on the events?  e.g. `zfs_slow_io_events_per_second_max`?  Or is that too verbose?

Presumably this does not limit the rate at which we count slow i/os for `zpool status`.  I wonder if it would be helpful to clarify that anywhere?